internal/conf/config.go

```diff
@@ -58,6 +58,9 @@ type OperatorConfig struct {
 
 	// WorkerOSBuildVersion is the release tag of OSBuild for the osbuild RPM
 	WorkerOSBuildVersion string `envconfig:"WORKER_OSBUILD_VERSION" default:"v59"`
+
+	// BaseISOContainerImage is the container image to run the iso-package job
+	BaseISOContainerImage string `envconfig:"BASE_ISO_CONTAINER_IMAGE" required:"true" default:"controller:latest"`
```

Comment:
The problem with this default value is that it is always incorrect.

You should set the environment variable to the name of the container image. See https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/#use-container-fields-as-values-for-environment-variables for information